packages/services/src/testutils.ts

```diff
@@ -774,7 +775,8 @@ export class FakeUserManager extends BaseManager implements User.IManager {
   constructor(
     options: UserManager.IOptions = {},
     identity: User.IIdentity,
-    permissions: ReadonlyJSONObject
+    permissions: ReadonlyJSONObject,
+    updatableFields: User.UpdatableUserField[] = []
```

Comment:
I wonder, have you explored if this could be encoded as a permission instead of a new field? It feels like user would be granted permission to change its avatar, or nickname so I would be just wary of duplicating concepts; if there are good reasons not to use permissions then its fine to add it as-is.

Reply:
I updated https://github.com/jupyter-server/jupyter_server/pull/1518 with this change.
I converted this PR to draft, waiting for https://github.com/jupyter-server/jupyter_server/pull/1518 to be merged, then I'll close it.